index.md

```diff
@@ -14,12 +14,13 @@ SoCraTes UK is totally community-focused. It's a great opportunity to speak to a
 
 ### Registration and Price Information
 
-Add yourself to the Socrates UK 2018 Waiting List to get the opportunity to purchase either a Single Room or Twin Room ticket. Please note that tickets will be allocated on a random basis to ensure that as many people as possible will have the opportunity to attend. We’re aiming to release tickets to the Waiting List early March, so keep an eye out for our announcement email and on Twitter via [@SoCraTes_UK][@socrates_uk]
+The details of SoCraTes UK 2019 are currently being worked out. Once the date and price has been confirmed, you will be able add yourself to the Waiting List to get the opportunity to purchase either a Single Room or Twin Room ticket. Please note that tickets will be allocated on a random basis to ensure that as many people as possible will have the opportunity to attend.
```

Comment:
could we please lowercase "waiting list", "single room" and "twin room"?

Reply:
👍